.github/workflows/next-rc.yml

```diff
@@ -0,0 +1,104 @@
+name: Prepare next release candidate
```

Comment:
I was not able to do the patch release: https://github.com/mszostok/botkube/actions/runs/3135175021/jobs/5090553564

I did a final release `v0.52.0` then I merge a new PR to `release-0.52` and the next RC failed job with:
```
v0.52.0 is already in remote, exiting...
```

Reply:
No worries, added better messaging.